.github/workflows/release.yml

```diff
@@ -203,8 +203,18 @@ jobs:
               --appdir "$APP.AppDir" -d contrib/Helix.desktop \
               -i contrib/helix.png --output appimage
 
-          mv "$APP-$VERSION-$ARCH.AppImage" \
-              "$APP-$VERSION-$ARCH.AppImage.zsync" dist
+          # anylinux appimage
+          APPIMAGETOOL=$(wget -q https://api.github.com/repos/probonopd/go-appimage/releases -O - \
```

Comment:
I'm not too keen on downloading and executing random binaries off of github as part of the release process. If a release is ever injected, it could modify the appimages we provided

Reply:
Hello @archseer by random binary you mean go-appimage? what makes it a random binary compared to linuxdeploy which is already downloaded and executed as well?

Reply:
Fair point, I'd prefer we don't do that either.

Reply:
Ok I just did that, now linuxdeploy nor go-appimage are being used to deploy the appimage.

however there is still a need for an appimagetool to turn the AppDir into an appimage, I guess this could also be done manually but that would mean embedding the update information in the squashfs image and I really don't know how to do that.